client/src/Movies/MovieCard.js

```diff
@@ -1,9 +1,11 @@
 import React from 'react';
 import { Link } from 'react-router-dom';
+// import axios from 'axios';
 
 const MovieCard = props => {
```

Comment:
you an destructure here: `const MovieCard = { title, director, metascore, stars } => {`